lib/content-services/breadcrumb/breadcrumb.component.spec.ts

```diff
@@ -205,4 +205,26 @@ describe('Breadcrumb', () => {
         expect(route[0].id).toBe('custom-id');
         expect(route[0].name).toBe('custom-name');
     });
+
+    it('should apply the transformation function when there is one', () => {
```

Comment:
The parameter of transform (node) method is not tested this way. In the other test file the parameter is not needed to be tested, but here, I think it would be good.

Reply:
🥇